cardea-server/src/main/resources/application.yml

```diff
@@ -17,6 +17,8 @@ spring:
 server:
   use-forwarded-headers: true
   forward-headers-strategy: framework
+  error:
+    include-stacktrace: on_param
```

Comment:
Maybe useful in a local application.properties for development, but stack traces shouldn't be returned in production

Reply:
Exactly. We include `spring-boot-devtools` in the server pom to aid in development, and by default it included the stack trace in the response to the user, which I wanted to avoid. Setting this to `on_param` ensures it's turned off by default unless enabled with a URL query param, which we might want to do in development (but our users will never do in production).

Reply:
I could also set this to `never` and then developers could override it in their local `application.properties` file.

Reply:
Ah, thanks for the explanation. I didn't realize this and should set it up the same on Dimsum